src/devtools/client/debugger/src/components/PrimaryPanes/index.tsx

```diff
@@ -60,7 +60,6 @@ const mapStateToProps = (state: UIState) => {
   return {
     cx: getContext(state),
     selectedTab: getSelectedPrimaryPaneTab(state),
-    sourceSearchOn: getActiveSearch(state) === "source",
```

Comment:
:speech_balloon: TS pointed out that a comparison with this field's possible values would never match, and it looks like this prop isn't being used any more